udata_gouvfr/theme/templates/subnav-small.html

```diff
@@ -13,25 +14,14 @@
                 {% set action = url_for('front.search') %}
                 {% set placeholder = _('Search') %}
             {% endif %}
-            <form action="{{ action }}"
-                class="col-sm-4 col-md-6 col-lg-6 col-xs-12">
-                {% if in_url('sort') and in_url('organization') %}
-                    <input type="hidden" name="sort" value="{{ request.args.sort }}">
-                    <input type="hidden" name="organization" value="{{ request.args.organization }}">
-                {% elif territory %}
-                    <input type="hidden" name="geozone" value="{{ territory.id }}">
-                {% endif %}
-                <div class="form-group">
-                    <div class="input-group">
-                        <div class="input-group-btn">
-                            <button class="btn" type="submit"><i class="fa fa-search"></i></button>
-                        </div>
-                        <input id="main-search" name="q" type="search" class="form-control"
-                            autocomplete="off" placeholder="{{ placeholder }}"
-                            value="{{ request.args.q }}">
-                    </div>
-                </div>
-            </form>
+            <div class="col-sm-4 col-md-6 col-lg-6 col-xs-12">
+                <site-search v-ref:search
+                    action="{{ action }}"
+                    placeholder="{{ placeholder }}"
+                    {% if territory %}geozone="{{ territory.id }}"{% endif %}
```

Comment:
Good point. I was wondering if the only usage was territory but I guess you're right, territory-id/territoryId is more explicit